pom.xml

```diff
@@ -12,9 +12,9 @@
   <version>5.4-SNAPSHOT</version>
   <packaging>hpi</packaging>
 
-  <name>CloudBees Folders Plugin</name>
+  <name>Folders Plugin</name>
   <description>This plugin allows users to create "folders" to organize jobs. Users can define custom taxonomies (like
-    by project type, organization type etc). Folders are nestable and you can define views within folders.
+    by project type, organization type etc). Folders are nestable and you can define views within folders. Maintained by CloudBees, Inc.
   </description>
   <url>https://wiki.jenkins-ci.org/display/JENKINS/CloudBees+Folders+Plugin</url>
```

Comment:
You mean, because you are too busy now? That is fine, just checking if it was planned.